tests/lsstcomcamsim/cdb_lsstcomcamsim.sql

```diff
@@ -0,0 +1,323 @@
+	group_id VARCHAR(64), 
+	cur_index INTEGER, 
+	max_index INTEGER, 
+	img_type VARCHAR(64), 
+	emulated BOOLEAN, 
+	science_program VARCHAR(64), 
+	observation_reason VARCHAR(68), 
+	target_name VARCHAR(64), 
+	air_temp FLOAT, 
+	pressure FLOAT, 
+	humidity FLOAT, 
+	wind_speed FLOAT, 
+	wind_dir FLOAT, 
+	dimm_seeing FLOAT, 
+	focus_z FLOAT, 
+	simulated BOOLEAN, 
+	vignette VARCHAR(10), 
+	vignette_min VARCHAR(10), 
+	s_region VARCHAR(1024), 
+	PRIMARY KEY (exposure_id), 
+	CONSTRAINT un_day_obs_seq_num UNIQUE (day_obs, seq_num)
+)
+
+;
+
+CREATE TABLE main.exposure_flexdata_schema (
+	"key" VARCHAR(128) NOT NULL, 
+	dtype VARCHAR(64) NOT NULL, 
+	doc TEXT NOT NULL, 
+	unit VARCHAR(128), 
+	ucd VARCHAR(128), 
+	PRIMARY KEY ("key")
+)
+
+;
+
+CREATE TABLE main.ccdexposure_flexdata_schema (
+	"key" VARCHAR(128) NOT NULL, 
+	dtype VARCHAR(64) NOT NULL, 
+	doc TEXT NOT NULL, 
+	unit VARCHAR(128), 
+	ucd VARCHAR(128), 
+	PRIMARY KEY ("key")
+)
+
+;
+
+CREATE TABLE main.visit1 (
+	visit_id BIGINT NOT NULL, 
+	exposure_name VARCHAR(20) NOT NULL, 
+	controller VARCHAR(1) NOT NULL, 
+	day_obs INTEGER NOT NULL, 
+	seq_num INTEGER NOT NULL, 
+	physical_filter VARCHAR(32), 
+	band VARCHAR(32), 
+	s_ra DOUBLE, 
+	s_dec DOUBLE, 
+	sky_rotation DOUBLE, 
+	azimuth_start FLOAT, 
+	azimuth_end FLOAT, 
+	azimuth FLOAT, 
+	altitude_start FLOAT, 
+	altitude_end FLOAT, 
+	altitude FLOAT, 
+	zenith_distance_start FLOAT, 
+	zenith_distance_end FLOAT, 
+	zenith_distance FLOAT, 
+	airmass FLOAT, 
+	exp_midpt TIMESTAMP, 
+	exp_midpt_mjd DOUBLE, 
+	obs_start TIMESTAMP, 
+	obs_start_mjd DOUBLE, 
+	obs_end TIMESTAMP, 
+	obs_end_mjd DOUBLE, 
+	exp_time FLOAT, 
+	shut_time FLOAT, 
+	dark_time FLOAT, 
+	group_id VARCHAR(64), 
+	cur_index INTEGER, 
+	max_index INTEGER, 
+	img_type VARCHAR(64), 
+	emulated BOOLEAN, 
+	science_program VARCHAR(64), 
+	observation_reason VARCHAR(68), 
+	target_name VARCHAR(64), 
+	air_temp FLOAT, 
+	pressure FLOAT, 
+	humidity FLOAT, 
+	wind_speed FLOAT, 
+	wind_dir FLOAT, 
+	dimm_seeing FLOAT, 
+	focus_z FLOAT, 
+	simulated BOOLEAN, 
+	vignette VARCHAR(10), 
+	vignette_min VARCHAR(10), 
+	s_region VARCHAR(1024), 
+	PRIMARY KEY (visit_id)
+)
+
+;
+
+CREATE TABLE main.ccdvisit1 (
+	ccdvisit_id BIGINT NOT NULL, 
+	visit_id BIGINT NOT NULL, 
+	detector INTEGER NOT NULL, 
+	s_region VARCHAR(1024), 
+	PRIMARY KEY (ccdvisit_id)
+)
+
+;
+
+CREATE TABLE main.exposure_flexdata (
+	obs_id BIGINT NOT NULL, 
+	"key" VARCHAR(128) NOT NULL, 
+	value TEXT, 
+	PRIMARY KEY (obs_id, "key"), 
+	CONSTRAINT fk_exposure_flexdata_obs_id FOREIGN KEY(obs_id) REFERENCES exposure (exposure_id), 
+	CONSTRAINT fk_exposure_flexdata_key FOREIGN KEY("key") REFERENCES exposure_flexdata_schema ("key")
+)
+
+;
+
+CREATE TABLE main.ccdexposure (
+	ccdexposure_id BIGINT NOT NULL, 
+	exposure_id BIGINT NOT NULL, 
+	detector INTEGER NOT NULL, 
+	s_region VARCHAR(1024), 
+	PRIMARY KEY (ccdexposure_id), 
+	CONSTRAINT un_exposure_id_detector UNIQUE (exposure_id, detector), 
```

Comment:
I think the `CONSTRAINT` names need to have the table name in them, in this case `ccdexposure`